### PR TITLE
calendar - update light and dark schemas according to new design

### PR DESCRIPTION
**Pull request by desig9stein**
closes https://github.com/IgniteUI/igniteui-webcomponents/issues/1263

### CODE REVIEW

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 1]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
 /// @prop {Map} date-disabled-foreground [color: ('gray', 500, .5)] - The foreground color of disabled dates.
 /// @prop {Map} date-disabled-range-foreground [color: ('gray', 900, .38)] - The foreground color of disabled dates.
-/// @prop {Map} date-current-border-color [color: ('primary', 400)] - The border color of the current date.
-/// @prop {Map} date-current-hover-border-color [color: ('primary', 300)] - The :hover border color of the current date.
-/// @prop {Map} date-current-focus-border-color [color: ('primary', 300)] - The :focus border color of the current date.
-/// @prop {Map} date-special-range-foreground [color: ('primary', 50)] - The foreground color of a special date in rage selection.
+/// @prop {Map} date-current-foreground [color: ('gray', 900)] - The foreground color of the current date.
+/// @prop {Map} date-current-background [color: ('primary', 700, .5)] - The background color of the current date.
+/// @prop {Map} date-current-hover-background [color: ('primary', 700)] - The :hover background color of the current date.
+/// @prop {Map} date-current-focus-background [color: ('primary', 700)] - The :hover background color of the current date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:463]:
```suggestion
/// @prop {Map} date-current-focus-background [color: ('primary', 700)] - The :focus background color of the current date.
```

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 2]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
 /// @prop {Map} date-special-range-background [color: ('primary', 600, .4)] - The background color of a special date in range selection.
-/// @prop {Map} date-special-foreground [color: ('primary', 50)] - The foreground color of a special date.
-/// @prop {Map} date-special-background [color: ('primary', 600, .4)] - The background color of a special date.
-/// @prop {Map} date-special-hover-foreground [contrast-color: ('primary', 600)] - The foreground color of a special date.
-/// @prop {Map} date-special-hover-background [color: ('primary', 400, .7)] - The background color of a special date.
-/// @prop {Map} date-special-focus-foreground [contrast-color: ('primary', 600)] - The foreground color of a special date.
-/// @prop {Map} date-special-focus-background [color: ('primary', 400, .7)] - The background color of a special date.
+/// @prop {Map} date-special-foreground [color: ('primary', 100)] - The foreground color of a special date.
+/// @prop {Map} date-special-hover-foreground [color: ('primary', 50)] - The foreground color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:471]:
```suggestion
/// @prop {Map} date-special-hover-foreground [color: ('primary', 50)] - The :hover foreground color of a special date.
```

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 3]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-hover-background [contrast-color: ('gray', 50, .1)] - The background color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:472]:
```suggestion
/// @prop {Map} date-special-hover-background [contrast-color: ('gray', 50, .1)] - The :hover background color of a special date.
```

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 4]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-focus-foreground [color: ('primary', 50)] - The foreground color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:473]:
```suggestion
/// @prop {Map} date-special-focus-foreground [color: ('primary', 50)] - The :focus foreground color of a special date.
```

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 5]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-focus-background [contrast-color: ('gray', 50, .1)] - The background color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:474]:
```suggestion
/// @prop {Map} date-special-focus-background [contrast-color: ('gray', 50, .1)] - The :focus background color of a special date.
```

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 6]
```diff
@@ -455,22 +455,28 @@ $dark-bootstrap-calendar: extend(
...
 /// @prop {Map} date-special-range-hover-background [color: ('primary', 600, .7)] - The :hover background color of a special date in range selection.
 /// @prop {Map} date-special-range-focus-background [color: ('primary', 600, .7)] - The :focus background color of a special date in range selection.
+/// @prop {Map} date-special-border-color [color: ('primary', 100)] - The outline color around a special date.
+/// @prop {Map} date-special-hover-border-color [color: ('primary', 50)] - The outline color around a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:478]:
```suggestion
/// @prop {Map} date-special-hover-border-color [color: ('primary', 50)] - The outline color around a special date on hover.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 7]
```diff
@@ -1871,20 +1913,26 @@ $bootstrap-calendar: extend(
...
 /// @prop {Map} date-special-foreground [color: ('primary', 500)] - The foreground color of a special date.
-/// @prop {Map} date-special-background [color: ('primary', 50, .7)] - The background color of a special date.
-/// @prop {Map} date-special-hover-foreground [color: ('primary', 700)] - The foreground color of a special date.
-/// @prop {Map} date-special-hover-background [color: ('primary', 50)] - The background color of a special date.
-/// @prop {Map} date-special-focus-foreground [color: ('primary', 700)] - The foreground color of a special date.
-/// @prop {Map} date-special-focus-background [color: ('primary', 50)] - The background color of a special date.
-/// @prop {Map} date-special-range-hover-background [color: ('primary', 50)] - The :hover background color of a special date in range selection.
-/// @prop {Map} date-special-range-focus-background [color: ('primary', 50)] - The :focus background color of a special date in range selection.
+/// @prop {Color} date-special-background [transparent] - The background color of a special date.
+/// @prop {Map} date-special-hover-foreground [color: ('primary', 600)] - The foreground color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1926]:
```suggestion
/// @prop {Map} date-special-hover-foreground [color: ('primary', 600)] - The :hover foreground color of a special date.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 8]
```diff
@@ -1871,20 +1913,26 @@ $bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-hover-background [color: ('gray', 900, .05)] - The background color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1927]:
```suggestion
/// @prop {Map} date-special-hover-background [color: ('gray', 900, .05)] - The :hover background color of a special date.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 9]
```diff
@@ -1871,20 +1913,26 @@ $bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-focus-foreground [color: ('primary', 600)] - The foreground color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1928]:
```suggestion
/// @prop {Map} date-special-focus-foreground [color: ('primary', 600)] - The :focus foreground color of a special date.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 10]
```diff
@@ -1871,20 +1913,26 @@ $bootstrap-calendar: extend(
...
+/// @prop {Map} date-special-focus-background [color: ('gray', 900, .05)] - The background color of a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1929]:
```suggestion
/// @prop {Map} date-special-focus-background [color: ('gray', 900, .05)] - The :focus background color of a special date.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 11]
```diff
@@ -1897,28 +1945,28 @@ $bootstrap-calendar: extend(
...
-/// @prop {Map} date-selected-current-range-hover-background [color: ('gray', 900, .05)] - The :hover background color of the current date in the selected range.
-/// @prop {Map} date-selected-current-range-focus-background [color: ('gray', 900, .05)] - The :focus background color of the current date in the selected range.
+/// @prop {Map} date-selected-current-range-hover-background [color: ('primary', 50)] - The :hover background color of the current date in the selected range.
+/// @prop {Map} date-selected-current-range-focus-background [color: ('primary', 50)] - The :focus background color of the current date in the selected range.
 /// @prop {Map} border-color [color: ('gray', 400)] - The calendar border color.
-/// @prop {Color} date-special-border-color [transparent] - The outline color around a special date.
-/// @prop {Color} date-special-hover-border-color [transparent] - The outline color around a special date.
+/// @prop {Map} date-special-border-color [color: ('primary', 500)] - The outline color around a special date.
+/// @prop {Map} date-special-hover-border-color [color: ('primary', 600)] - The outline color around a special date.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1961]:
```suggestion
/// @prop {Map} date-special-hover-border-color [color: ('primary', 600)] - The :hover outline color around a special date.
```

File: sass/themes/schemas/components/light/_calendar.scss [hunk 12]
```diff
@@ -1897,28 +1966,28 @@ $bootstrap-calendar: extend(
...
-/// @prop {Map} date-selected-current-range-focus-background [color: ('gray', 900, .05)] - The :focus background color of the current date in the selected range.
+/// @prop {Map} date-selected-current-range-hover-background [color: ('primary', 50)] - The :hover background color of the current date in the selected range.
+/// @prop {Map} date-selected-current-range-focus-background [color: ('primary', 50)] - The :focus background color of the current date in the selected range.
 /// @prop {Map} border-color [color: ('gray', 400)] - The calendar border color.
-/// @prop {Color} date-special-border-color [transparent] - The outline color around a special date.
-/// @prop {Color} date-special-hover-border-color [transparent] - The outline color around a special date.
+/// @prop {Map} date-special-border-color [color: ('primary', 500)] - The outline color around a special date.
+/// @prop {Map} date-special-hover-border-color [color: ('primary', 600)] - The :hover outline color around a special date.
 /// @prop {Map} ym-selected-current-hover-foreground [contrast-color: ('primary', 900)] - The :hover foreground color of the current/selected month/year.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/light/_calendar.scss:1983]: This property is duplicated - see line 1997

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 13]
```diff
@@ -668,6 +684,24 @@ $dark-indigo-calendar: extend(
...
+                100,
+            ),
+        ),
+        date-special-hover-border-color: (
+            color: (
+                'primary',
+                50,
+            ),
+        ),
+        date-special-range-border-color: (
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:699]: This one is not documented.

File: sass/themes/schemas/components/dark/_calendar.scss [hunk 14]
```diff
@@ -479,13 +485,11 @@ $dark-bootstrap-calendar: extend(
...
-/// @prop {Map} ym-current-outline-focus-color [color: ('primary', 300)] - The :focus outline color of the current month/year.
-/// @prop {Map} ym-current-outline-color [color: ('primary', 400)] - The outline color of the current month/year.
-/// @prop {Map} ym-selected-current-outline-color [contrast-color: ('gray', 50, .6)] - The outline color of the selected current month/year.
-/// @prop {Map} ym-selected-current-outline-hover-color [contrast-color: ('gray', 50, .6)] - The :hover outline color of the selected current month/year.
-/// @prop {Map} ym-selected-current-outline-focus-color [contrast-color: ('gray', 50, .6)] - The :focus outline color of the selected current month/year.
-/// @prop {Map} date-current-hover-background [contrast-color: ('gray', 50, .1)] - The :hover background color of the current date.
+/// @prop {Map} ym-current-background [color: ('primary', 700, .5)] - The background color of the current month/year.
+/// @prop {Map} ym-current-hover-background [color: (color: ('primary', 700)] - The :hover background color of the current month/year.
```
Comment (SisIvanova) [inline sass/themes/schemas/components/dark/_calendar.scss:489]:
```suggestion
/// @prop {Map} ym-current-hover-background [color: ('primary', 700)] - The :hover background color of the current month/year.
```

**Review by SisIvanova — APPROVED** [review]

**Review by simeonoff — APPROVED** [review]

**Pull request merged**